### DISCUSSION TITLE
how to parse the from address?

### DISCUSSION THREAD

**Discussion by yuzhongrong** [Chia-Network/chia-blockchain#6065] [Support]
this is json string of one  get_trasaction :

{
    "additions": [
        {
            "amount": 1010000000000,
            "parent_coin_info": "0x8013f0fde5ea0b02577f7bfe519ae8c8251257ad0583589f62d47c4fec9a43ef",
            "puzzle_hash": "0x7de8bf0cb746480503382ecc09248216f328281456172eb00579bfd0ef2dc846"
        },
        {
            "amount": 747999999998,
            "parent_coin_info": "0x8013f0fde5ea0b02577f7bfe519ae8c8251257ad0583589f62d47c4fec9a43ef",
            "puzzle_hash": "0xd33f6a28843b73d6301c6cc53882025cb6f6da16253a073bef5e98d6093ea744"
        }
    ],
    "amount": 1010000000000,
    "confirmed": true,
    "confirmed_at_height": 77246,
    "created_at_time": 1621096949,
    "fee_amount": 2000000000,
    "name": "0x505ee9497c1fbc6942323c2f60e9679915969f4b40c161f0b9e54ae2c05937bd",
    "removals": [
        {
            "amount": 760000000000,
            "parent_coin_info": "0x00a7f7de0b161eb5689c085782a93684e552ca9427c2afea6c2beac09e90ba22",
            "puzzle_hash": "0x27f47de5b656bc6a0c33a9cb9026fd0feabce7892991a21bc8613d1f618e59dc"
        },
        {
            "amount": 999999999998,
            "parent_coin_info": "0xecc44c2fb760e35ef179deeec7556540e1698add1ccdb72c4ea9ccdc81d7f4f6",
            "puzzle_hash": "0xad83d4d02b40b2ba09e1bca08ac6bd2d2e611d84f30b056f6d8713eed3bc10c6"
        }
    ],
    "sent": 1,
    "sent_to": [
        [
            "ccbd6e89cde0e01f71f2c21369ce6dab795eb1bd8d5d95e5c1a4d3d0bc86ffcb",
            1,
            null
        ]
    ],
    "spend_bundle": {
        "aggregated_signature": "0x9319f08c8b4254d2f1288480c7c0755479b098ac82e3ad03b2b4a5f468391334a56f56e3723dc288b28f006ef77bd4cc04a2312cc14cdf626ca0e69c2966130cabdafb2fdb31766469123c2289d6eeb8ced6cb210f3aa91f2ff74a56f5fd9d28",
        "coin_solutions": [
            {
                "coin": {
                    "amount": 760000000000,
                    "parent_coin_info": "0x00a7f7de0b161eb5689c085782a93684e552ca9427c2afea6c2beac09e90ba22",
                    "puzzle_hash": "0x27f47de5b656bc6a0c33a9cb9026fd0feabce7892991a21bc8613d1f618e59dc"
                },
                "puzzle_reveal": "0xff02ffff01ff02ffff01ff02ffff03ff0bffff01ff02ffff03ffff09ff05ffff1dff0bffff1effff0bff0bffff02ff06ffff04ff02ffff04ff17ff8080808080808080ffff01ff02ff17ff2f80ffff01ff088080ff0180ffff01ff04ffff04ff04ffff04ff05ffff04ffff02ff06ffff04ff02ffff04ff17ff80808080ff80808080ffff02ff17ff2f808080ff0180ffff04ffff01ff32ff02ffff03ffff07ff0580ffff01ff0bffff0102ffff02ff06ffff04ff02ffff04ff09ff80808080ffff02ff06ffff04ff02ffff04ff0dff8080808080ffff01ff0bffff0101ff058080ff0180ff018080ffff04ffff01b098d0fedc1ec32f3d2508ee979f81820d6537442b9b8d08a720261d35b7539410791b5b4782ac72d56a58db3b37e0e75fff018080",
                "solution": "0xff80ffff01ffff33ffa07de8bf0cb746480503382ecc09248216f328281456172eb00579bfd0ef2dc846ff8600eb28b0f40080ffff33ffa0d33f6a28843b73d6301c6cc53882025cb6f6da16253a073bef5e98d6093ea744ff8600ae284637fe80ffff34ff847735940080ffff3cffa0dd294ea7b8b1df201b82063209da055b5a4f7d2be153d00bf25334f53e6eac798080ff8080"
            },
            {
                "coin": {
                    "amount": 999999999998,
                    "parent_coin_info": "0xecc44c2fb760e35ef179deeec7556540e1698add1ccdb72c4ea9ccdc81d7f4f6",
                    "puzzle_hash": "0xad83d4d02b40b2ba09e1bca08ac6bd2d2e611d84f30b056f6d8713eed3bc10c6"
                },
                "puzzle_reveal": "0xff02ffff01ff02ffff01ff02ffff03ff0bffff01ff02ffff03ffff09ff05ffff1dff0bffff1effff0bff0bffff02ff06ffff04ff02ffff04ff17ff8080808080808080ffff01ff02ff17ff2f80ffff01ff088080ff0180ffff01ff04ffff04ff04ffff04ff05ffff04ffff02ff06ffff04ff02ffff04ff17ff80808080ff80808080ffff02ff17ff2f808080ff0180ffff04ffff01ff32ff02ffff03ffff07ff0580ffff01ff0bffff0102ffff02ff06ffff04ff02ffff04ff09ff80808080ffff02ff06ffff04ff02ffff04ff0dff8080808080ffff01ff0bffff0101ff058080ff0180ff018080ffff04ffff01b08034133b4ae599b7bfe0cc7bad7be284807b4cef05d8ca6919df60d7bd4e74be2b3b4487fbab7ab35d435447d07249b1ff018080",
                "solution": "0xff80ffff01ffff3dffa03b776ea3fc31957cf8af1a6d4b323202b252e3b4cf018f7f993ed27a028390cc8080ff8080"
            }
        ]
    },
    "to_puzzle_hash": "0x7de8bf0cb746480503382ecc09248216f328281456172eb00579bfd0ef2dc846",
    "trade_id": null,
    "type": 1,
    "wallet_id": 1
}

**Comment by yuzhongrong** [comment]
i want to parse from address  by  above json string

**Comment by BrandtH22** [comment]
@yuzhongrong , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - the chia client loads this json response as a dict allowing for standard python dict parsing, if you use the related rpc call you can parse it like any other json object 👍 

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
